grid-proxy/cmds/proxy_server/main.go

```diff
@@ -129,7 +131,7 @@ func main() {
 		log.Fatal().Err(err).Msg("failed to create GPU indexer")
 	}
 
-	indexer.Start(ctx)
+	indexer.Start(ctx, dbConnStr)
```

Comment:
the indexer, can already insert to the database, why is this passed?

Reply:
https://github.com/threefoldtech/tfgrid-sdk-go/blob/development/grid-proxy/internal/gpuindexer/gpuindexer.go#L23